shared/src/iosMain/kotlin/com/twilio/passkeys/TwilioPasskey.kt

```diff
@@ -68,119 +57,21 @@ internal enum class Attachment(val value: String) {
  * Represents the Twilio Passkey class responsible for managing passkey operations.
  *
  * @property passkeyPayloadMapper The passkey payload mapper used for mapping passkey payloads and responses.
- * @property deviceUtils The utility class for device-related operations.
  */
-actual class TwilioPasskey private constructor(
+actual class TwilioPasskey internal constructor(
```

Comment:
Can `mapToTwilioException` be part of this class instead of the wrapper? As it is a TwilioException, this could be more related to this class and the wrapper can pass the original error